### ISSUE TITLE
Missing certs/smime.pem cause tests to fail

### ISSUE THREAD

**Issue by timlegge** [dsully/perl-crypt-openssl-x509#89]
# Issue Template

## Description
make test fails with:

ok 56 - serial()
Crypt::OpenSSL::X509: Failed to create BIO at t/x509.t line 102.

ok($x509 = Crypt::OpenSSL::X509->new_from_file('certs/smime.pem'), 'new_from_file()');

certs/smime.pem dioes not exist

### Expected behaviour

run without error

### Actual behaviour

make test fails

## Operating system and version

Alpine edge 5.34 perl
### Windows (pre version 7)

`C:\> ver`

### Windows (Version 7 and newer)

`C:\> systeminfo | findstr /B /C:"OS Name" /C:"OS Version"`

### Linux and Unix based systems

`$ uname -a`

_For information on the specific distribution. Please consult your distribution documentation._

## Crypt::OpenSSL::X509 version

_Please provide version on the Crypt::OpenSSL::X509 version used_

_If working in a fork of the GitHub project, obtain this information post installation using:_

```bash
$ perl -I lib -MCrypt::OpenSSL::X509 \
-e 'print "$Crypt::OpenSSL::X509::VERSION\n"'
```

_If you are working on code, which has diverged from the lastest release, please mention this_.

## Perl version

_Please provide version on the Perl version used._

`$ perl -V`

## OpenSSL version

_Please provide version on the OpenSSL version used._

`$ openssl version`

## Output, if available

_Please add output, which can be used to describe the issue or circumstances surrounding the issue._

## Step by step guide to reproducing the issue

_If possible a code snipppet or unit-test to demonstrate the issue or step by step description of how to get reproduce the error._

_If the issue is related to a broken build please provide information on your build environment, like compiler, compiler version, parameters and environment_

**Comment by timlegge** [comment]
you need to:

make mainfest 

prior to the release

**Comment by jonasbn** [comment]
Thanks @timlegge 

I have merged your PR for Windows testing using GitHub Actions also (#85), it would not have caught this issue, but it will at least help me with potentional Windows issues.

**Comment by jonasbn** [comment]
Fixed in version 1.9.10, just uploaded to PAUSE/CPAN
